net/kea/Makefile

```diff
@@ -0,0 +1,198 @@
+#
+# Copyright (C) 2019 Banglang Huang <banglang.huang@foxmail.com>
+# Copyright (C) 2019 Rosy Song <rosysong@rosinson.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=kea
+PKG_VERSION:=460-update-mysql-database-schema-for-dhcpv6-config-backend--port_base
+PKG_RELEASE:=1
+PKG_MAINTAINER:=BangLang Huang<banglang.huang@foxmail.com> \
+  Rosy Song<rosysong@rosinson.com>
+PKG_BUILD_DEPENDS:=kea/host
+# real dir name in released tar.gz file
+RELEASE_NAME:=460-update-mysql-database-schema-for-dhcpv6-config-backend-port_base
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(RELEASE_NAME)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)-$(RELEASE_NAME)
+
+PKG_CAT:=zcat
+PKG_SOURCE:=$(PKG_VERSION)
+PKG_SOURCE_URL:=https://codeload.github.com/isc-projects/kea/tar.gz/
+PKG_HASH:=0951e513e8907a58ebde61a0a7005a71d22c6bd4270390f0dbb050f28847a054
+
+PKG_INSTALL:=1
+PKG_LICENSE:=MPL-2.0
+PKG_FIXUP:=autoreconf
```

Comment:
As this is using autotools, might want to add PKG_BUILD_PARALLEL for faster compilation.